src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import "../utils/mixin.scss";
+@import "./variables.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
it's normal by default

Reply:
so you can remove it
<img width="716" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/cf8a4bfd-e341-4914-bfb0-4bf151ed6fe1">